### DISCUSSION TITLE
Why I've been verified as a student for 2 days but cannot use free copilot yet?

### DISCUSSION THREAD

**Discussion by muenyz** [community/community#65277] [GitHub Education] [answered]
### Hi everyone,

Is the approved submission invalid or shall I just keep waiting?What's more I didn't receive any email telling success, only received some telling failures.
![1](https://github.com/community/community/assets/143328867/12af8db7-368a-49df-b256-8fb67cbbb88b)
![2](https://github.com/community/community/assets/143328867/12848d72-af40-4640-9608-440f3e7bc04a)
![3](https://github.com/community/community/assets/143328867/27e36ac2-3875-466a-93b2-021f83821f2d)

**Comment by arafajs** [comment]
It may take a few days for your student verification to be processed. Once you have been verified, it can take a few days for the benefits of the Student Developer Pack to be reflected in your account. This is because GitHub has to manually review your verification request.

**Reply by muenyz** [reply]
Thanks.I got it.By the way,how long does it generally take after I have been verified?

**Reply by arafajs** [reply, marked as answer]
@muenyz 
It generally takes about 4 days for students to get access to GitHub Copilot after they have been verified. However, it may take longer depending on the volume of verification requests that GitHub is receiving.

**Reply by muenyz** [reply]
OK!Thank you very much.

**Reply by muenyz** [reply]
But is it normal that My friend gained access to free copilot as soon as he was verified today while I was verified earlier but still cannot use free copilot?

**Reply by muenyz** [reply]
Now solved,thanks.

**Reply by Gin-Chuang** [reply]
How did u solve it at the end?

**Reply by Nauman-Mansoori** [reply]
My problem is exact same 2 times rejected and then approved and till yet i don't t get email regarding this .
Also I don't know when and how i will know that i had get my pack.

**Reply by julord87** [reply]
> Now solved,thanks.

How you fix this? I has been approved today and I still cannot use copilot for free.

**Reply by samus-aran** [reply]
Hi @julord87 👋 

Is the Approved status purple? If it isn't (it's green) then you still need to wait the 72 hours (but it may be longer).

**Comment by muenyz** [comment]
I just waited for a few more days.&nbsp;


Muenyz
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年10月18日(星期三) 下午2:46
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [community/community] Why I&#39;ve been verified as a student for 2 days but cannot use free copilot yet? (Discussion #65277)





 
How did u solve it at the end?

**Comment by echoandzoey** [comment]
I have verified and can use it a few weeks ago. BUT now it not work for me again TOT How can i do

**Reply by zainabzafari** [reply]
how long it will take to be verified?

**Comment by Imadnajam** [comment]
Confirm that you meet all the eligibility criteria for free access to Copilot as a student. GitHub may have specific requirements or conditions that need to be met

**Comment by wolf1994722** [comment]
it has verified 5 day

**Comment by Nauman-Mansoori** [comment]
Can you please describe it what you are saying

**Comment by tinmale** [comment]
what should i do if i already pass the application but i just start a free copilot pride?

**Comment by Marek21SK** [comment]
I am waiting 6 days but still nothing "We're still processing your application. Once this is complete you will have access to the student developer pack offers."

**Comment by samus-aran** [comment]
We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
